our_repo.py

```diff
@@ -80,7 +80,52 @@ def number_of_016_files(self):
 
         return elm_016_count
 
+    def get_files_for_017(self, pattern):
+        pattern = pattern.replace('.', '/')
+        all_files = glob.glob(f'{self.repo_dir}/**/{pattern}.elm', recursive=True)
+
+        return [ file for file in all_files
```

Comment:
:art: I usually try to avoid using `file` as a variable name, but not a big concern here since it's localized within the list comprehension. (seeing it syntax highlighted is visually distracting a bit though)